examples/no-std/Cargo.lock

```diff
@@ -292,13 +292,39 @@ dependencies = [
  "paste",
  "radix-engine-common",
  "radix-engine-derive",
+ "regex",
  "sbor",
  "scrypto-schema",
  "serde_json",
  "strum",
  "utils",
 ]
 
+[[package]]
+name = "regex"
```

Comment:
Do these need to be included now?

Reply:
Yes, it's part of `radix-engine-interface` for wider reusability.